src/application/components/Explorer/GraphRefModal.tsx

```diff
@@ -29,207 +33,88 @@ export const GraphRefModal = ({
   const [graphVariant, setGraphVariant] = useState<string>();
 
   return (
-    <>
-      <div
-        css={{
-          width: "100vw",
-          height: "100vh",
-          position: "absolute",
-          backgroundColor: "rgb(25, 28, 35)",
-          opacity: "0.55",
-        }}
-      ></div>
-      <div
-        css={{
-          width: 470,
-          height: 400,
-          position: "absolute",
-          margin: "auto",
-          top: "20%",
-          left: "calc(50% - 250px)",
-          display: "flex",
-          backgroundColor: "white",
-          borderRadius: 12,
-          boxShadow:
-            "rgb(0 0 0 / 12%) 0px 16px 32px 0px, rgb(18 21 26 / 4%) 0px 0px 0px 1px",
-          fontFamily: '"Source Sans Pro", sans-serif',
-          fontSize: 15,
-          textAlign: "center",
-        }}
+    <Modal open onClose={onClose} size="lg" className="h-[422px]">
+      <Modal.Header
+        className={clsx("flex-col", { hidden: newGraphRefLoading })}
       >
-        <div css={{ display: "block", margin: "auto" }}>
-          {newGraphRefLoading ? (
+        <Telescope className="w-[125px] h-[100px] m-auto" />
+        <Modal.Title>
+          {wasTriggeredByIntrospectionFailure
+            ? "We could not introspect your schema."
+            : "Choose another graph to populate the Explorer."}
```

Comment:
In practice, these two states will never be shown together since the header is not displayed when the modal is in loading state: https://github.com/apollographql/apollo-client-devtools/blob/ad711ed7aba99f824da9d36ce90485704099b519/src/application/components/Explorer/GraphRefModal.tsx#L37-L39

<img width="485" alt="Screenshot 2024-01-24 at 2 52 00 PM" src="https://github.com/apollographql/apollo-client-devtools/assets/565661/ff0000c7-8ae8-4fac-8ebc-8870e0d053c3">

We should be fine 🙂. 

I did fix the spinner background though.